src/collections/_documentation/performance/distributed-tracing.md

```diff
@@ -412,7 +412,20 @@ $ npm install @sentry/browser
 $ npm install @sentry/apm
 ```
 
-#### Sending Traces/Transactions/Spans
+OR
+
+Alternativly if you can't use npm packages you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/apm`:
```

Comment:
```suggestion
Alternatively, instead of npm packages, you can use our pre-built CDN bundle that combines both `@sentry/browser` and `@sentry/apm`:
```

- Both "OR" and "Alternatively" express alternative, we can use just one
- Users may choose to use the CDN even if they "can use npm", those are not mutually exclusive options